HomeTasks/HW3.py

```diff
@@ -1,14 +1,14 @@
 #TASK 1
-# text = input('Enter the text: ')
-#
-# for letter in text:
-#     if letter.isdigit():
-#         if int(letter) % 2 == 0:
-#             print(f"{letter} even numer")
-#         else:
-#             print(f"{letter} not even numer")
-#     else:
-#         print(f"{letter}")
+text = input('Enter the text: ')
+
+for letter in text:
+    if letter.isdigit():
+        if int(letter) % 2 == 0:
+            print(f"{letter} even numer")
+        else:
+            print(f"{letter} not even numer")
+    else:
```

Comment:
You don't have a condition for symbols. You can check if the letter is alphabetical (.isalpha()) for case with letters and all other cases will be symbols